src/taskmanager.cpp

```diff
@@ -104,7 +104,8 @@ void TaskManager::backupNow(BackupTaskPtr backupTask)
     }
     bTask->setCommand(makeTarsnapCommand(CMD_TARSNAP));
     bTask->setArguments(args);
-    backupTask->setCommand(bTask->command() + " " + bTask->arguments().join(" "));
+    backupTask->setCommand(bTask->command() + " " +
```

Comment:
Unfortunately that's stored as a string, not a StringList, so it would need more sophisticated code to split it into argument.  The same applies to archives created outside of the GUI.

How about we don't try to escape the archive creation commands?  It's much less likely that anybody would try to copy&paste those commands -- even if they were escaped, the command would still fail because you can't have two archives with the same name.